api/server/router/system/system_routes.go

```diff
@@ -89,51 +100,89 @@ func (s *systemRouter) getVersion(ctx context.Context, w http.ResponseWriter, r
 	return httputils.WriteJSON(w, http.StatusOK, info)
 }
 
+// DiskUsageObject represents an object type used for disk usage query filtering.
+type DiskUsageObject string
+
+const (
+	// ContainerObject represents a container DiskUsageObject.
+	ContainerObject DiskUsageObject = "containers"
+	// ImageObject represents an image DiskUsageObject.
+	ImageObject DiskUsageObject = "images"
+	// VolumeObject represents a volume DiskUsageObject.
+	VolumeObject DiskUsageObject = "volumes"
+	// BuildCacheObject represents a build-cache DiskUsageObject.
+	BuildCacheObject DiskUsageObject = "builds"
```

Comment:
Perhaps `build-cache` instead of `build` for this one 🤔

Reply:
The reason I went for `build` originally is because we also have e.g. `/build/*` routes responsible for this object type, along with `/images/*`, `/volumes/*` etc.
IMO `build` is more consistent in this case, do you disagree?

Reply:
Let me ping @tonistiigi if he has preference on this.

My main train of thought was; "can we expect other build-related objects at some point?" (idk, "background build jobs" or so) and in that case, would `build` (instead of `build-cache`) be the right term.


Naming is hard 😂